ovos_persona_server/metadata.py

```
@@ -0,0 +1,234 @@
+"""
+SQLAlchemy database setup and ORM models for the OVOS Persona Server.
+
+This module defines the asynchronous SQLAlchemy engine, session management,
+and declarative base for defining ORM models. It also includes the ORM
+definitions for 'files', 'vector_stores', and 'vector_store_files' tables,
+mirroring the OpenAI API's data structures.
+"""
+
+import json
+import os
+from typing import AsyncGenerator, Dict, Any, Optional, Union, List
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import

(F401)